modules/tfm/tfm/boards/common/tfm_hal_platform.c

```diff
@@ -27,20 +31,22 @@ enum tfm_hal_status_t tfm_hal_platform_init(void)
 #if defined(TFM_PARTITION_CRYPTO)
 	int err;
 
+#if !defined(TFM_CRYPTO_RNG_MODULE_DISABLED)
 	/* Initialize the nrf_cc3xx runtime */
```

Comment:
When this comment is within this ifdef it sounds like the runtime is conditionally initialized, but it is unconditional, so it should be one line higher.